docs/blockchain/contracts/nft-contract/nft-actions/stofrcfg.a.experimental.md

```diff
@@ -0,0 +1,70 @@
+---
+title: 'stofrcfg.a'
+order: 36
+
+---
+
+# stofrcfg.a
+
+Set global uniq offer configuration
+
+## Behavior
+
+Set global configuration for all uniq offer include both token and token factory.
```

Comment:
Should be
Set global configurations for all Uniq offers that will be made on either a token or a token factory.